### PR TITLE
[tests-only][full-ci] add tests to list space file shared with them when sync is disabled

### PR DESCRIPTION
**Pull request by nabim777**
## Description

In this PR following scenarios are added related to sharingNG:
```
- Scenario: user lists the space file shared with them when sync is disabled
- Scenario: user lists the space folder shared with them when sync is disabled
```

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/8318

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 1]
```diff
@@ -2974,4 +2974,514 @@ Feature: an user gets the resources shared to them
...
+      | resource        | textfile.txt |
+      | space           | Personal     |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Viewer       |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+    """
+      {
```
Comment (PrajwolAmatya) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:2995]:
```suggestion
    And the JSON data of the response should match
      """
      {
```

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 2]
```diff
@@ -2974,4 +2974,514 @@ Feature: an user gets the resources shared to them
...
+
+
+  Scenario: user lists the space folder shared with them when auto-sync is disabled
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has created a folder "folderToShare" in space "projectSpace"
+    And user "Brian" has disabled the auto-sync share
+    And user "Alice" has sent the following share invitation:
+      | resource        | folderToShare |
+      | space           | Personal      |
```
Comment (grgprarup) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:3257]:
```suggestion
      | space           | projectSpace      |
```

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 3]
```diff
@@ -2974,4 +2974,514 @@ Feature: an user gets the resources shared to them
...
+
+
+  Scenario: user lists the space file shared with them when auto-sync is disabled
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "to share" to "textfile.txt"
+    And user "Brian" has disabled the auto-sync share
+    And user "Alice" has sent the following share invitation:
+      | resource        | textfile.txt |
+      | space           | Personal     |
```
Comment (grgprarup) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:2987]:
```suggestion
      | space           | projectSpace     |
```

Comment (saw-jan) [reply tests/acceptance/features/apiSharingNg/sharedWithMe.feature:2987]: tests are passing even when the space is wrong? :thinking: 
Maybe json scema assertion is not working as expected?

Comment (nabim777) [reply tests/acceptance/features/apiSharingNg/sharedWithMe.feature]: Oh! I forget that step

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 4]
```diff
@@ -2974,4 +2974,435 @@ Feature: an user gets the resources shared to them
...
       }
     }
-    """
\ No newline at end of file
+    """
+
+
+  Scenario: user lists the space file shared with them when auto-sync is disabled
+    Given using spaces DAV path
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
```
Comment (S-Panta) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:2982]:
```suggestion
    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
```

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 5]
```diff
@@ -2974,4 +2974,435 @@ Feature: an user gets the resources shared to them
         }
       }
     }
-    """
\ No newline at end of file
+    """
+
+
+  Scenario: user lists the space file shared with them when auto-sync is disabled
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:2980]: indicate that the scenario is for project space
```suggestion
  Scenario: user lists the space file shared with them when auto-sync is disabled
```

**Review by saw-jan — CHANGES_REQUESTED** [review]
After https://github.com/owncloud/ocis/pull/8443 is merged

File: tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature [hunk 6]
```diff
@@ -1034,3 +1034,462 @@ Feature: listing sharedWithMe when auto-sync is disabled
         }
       }
       """
+
+
+  Scenario: user lists the space file shared with them when auto-sync is disabled
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature:1039]:
```suggestion
  Scenario: user lists file shares shared with them from project-space when auto-sync is disabled
```

**Review by saw-jan — APPROVED** [review]
LGTM

**Review by amrita-shrestha — APPROVED** [review]
lgtm

**Pull request merged**